### PR TITLE
Hiding the failure options from the training outcome flow

### PR DESCRIPTION
**Pull request by jesseyuen**
This PR:

- Skips the question: "Have they met the standards for QTS?"
- Removes the outcome row from the summary card
- Updates to content in the check answers page

### CODE REVIEW

File: app/views/record.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
           {% if hasCommencementDate%}
             {{ govukButton({
               text: "Record training outcome",
```
Comment (EmmaFrith) [inline app/views/record.html]: text: Recommend trainee for QTS

File: app/views/record/qts/outcome-date.html [hunk 2]
```diff
@@ -2,13 +2,15 @@
...
+{# {% set pageHeading %}
   {% if data.record.qtsDetails.standardsAssessedOutcome == "Yes" %}
     When did they meet the standards? 
   {% else %} 
     When should they be recorded as having not met the standards?
   {% endif %}
-{% endset %}
+{% endset %} #}
+
+{% set pageHeading = 'When did they meet the standards?' %}
```
Comment (EmmaFrith) [inline app/views/record/qts/outcome-date.html]: {% set pageHeading = 'When did they meet the QTS standards?' %}

File: app/views/record.html (hunk 1 above)
Comment (EmmaFrith) [reply app/views/record.html]: I think the button should be more focussed on the action that you can now do

File: app/views/record/qts/outcome-date.html (hunk 2 above)
Comment (EmmaFrith) [reply app/views/record/qts/outcome-date.html]: To be clear what standards we're referring to

Comment (EmmaFrith) [reply app/views/record/qts/outcome-date.html]: alternatively/ in addition - you could make the caption text more descriptive, i.e. 'Recommend Ameline Rodriguez for QTS'

**Review by edwardhorsford — CHANGES_REQUESTED** [review]
The main CTA should probably be reverted to say 'Recommend for QTS' or similar.

**Review by edwardhorsford — APPROVED** [review]

File: app/views/record.html (hunk 1 above)
Comment (jesseyuen) [reply app/views/record.html]: Good shout. Ta.

**Pull request merged**